specification/mediaservices/resource-manager/Microsoft.Media/stable/2020-05-01/streamingservice.json

```diff
@@ -292,13 +292,60 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Media/mediaservices/{accountName}/liveEvents/{liveEventName}/allocate": {
+      "post": {
+        "tags": [
+          "LiveEvents"
+        ],
+        "summary": "Allocate resources for a live event",
+        "description": "A live event is in StandBy state after allocation completes, and is ready to start.",
```

Comment:
why upper case on StandBy ? is that the state enumeration name? 
Maybe more details on why a customer should call allocate and the benefits of this state in improving startup speed and reducing costs for stand by hot pools of live events.

Reply:
Yes, it's using the actual enumeration state. It's unfortunate that we used incorrect casing for the enumeration and I didn't catch it in time.  Is it too late to change this now?  I think the additional information can be contained in its own separate documentation and we can add a reference to a fwlink to that in here..